moveit_kinematics/ikfast_kinematics_plugin/templates/CMakeLists.txt

```diff
@@ -8,29 +8,25 @@ find_package(catkin REQUIRED COMPONENTS
   pluginlib
   roscpp
   tf2_kdl
+  tf2_eigen
+  eigen_conversions
+  moveit_ros_planning
```

Comment:
moveit_ros_planning loads plugins from moveit_kinematics, so this is nearly a circular dependency. have you thought about this?

Reply:
That's something @mlautman introduced and I complained about it in https://github.com/ros-planning/moveit/pull/1014#pullrequestreview-165346664. However, that time it was neccessary for rdf_loader.
Using the new KinematicsBase API it became obsolete. Will change.